### PR TITLE
Shorten the title and description of the Bat Weather Project

### PR DESCRIPTION
**Pull request by FXF48-Mila**

### CODE REVIEW

**Comment by FXF48-Mila** [comment]
Hi @leewujung, following Aditya's suggestion, I shortened the project description to make it look better on the website. Could you approve the change?

File: content/project/2024-ubna-weather/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 # Documentation: https://sourcethemes.com/academic/docs/managing-content/
-title: "Analyzing the effects of different environmental conditions on bat activity"
+title: "Analyzing effects of different environmental conditions on bat activity"
```
Comment (leewujung) [inline content/project/2024-ubna-weather/index.md:3]: I guess I have a different suggestion 😅
```suggestion
title: "Analyzing the effects of environmental conditions on bat activity"
```

**Comment by leewujung** [comment]
@FXF48-Mila : Looks good! Could you remove the funding agency line from the bottom? Also, "the re-emergence of El Niño" feels a bit weird, since it is not like it hasn't occurred for a while. I suggest reading a bit more about it and find another way to describe it.

**Comment by FXF48-Mila** [comment]
Hi @leewujung, thank you for your detailedcomments. I've refined the description and rewritten part of the opening sentence in the first paragraph about the environmental changes. Please let me know if there are any other changes that need to be made. Thanks!

**Comment by leewujung** [comment]
Thanks @FXF48-Mila ! Merging now.

**Pull request merged**